app/models/form_profiles/va_526ez.rb

```diff
@@ -211,24 +210,6 @@ def convert_vets360_address(address)
     }.compact
   end
 
-  # Convert PCIU address to a Common address type
-  def get_common_address
-    service = EVSS::PCIUAddress::Service.new(user)
-    response = service.get_address
-    case response.address
-    when EVSS::PCIUAddress::DomesticAddress
-      prefill_domestic_address(response.address)
-    when EVSS::PCIUAddress::InternationalAddress
-      prefill_international_address(response.address)
-    when EVSS::PCIUAddress::MilitaryAddress
-      prefill_military_address(response.address)
-    else
-      {}
-    end
-  rescue
-    {}
-  end
-
```

Comment:
get_common_address is not called anywhere in Vets-API. This will not break anything.